src/components/UnitInput/__stories__/index.stories.tsx

```diff
@@ -0,0 +1,100 @@
+import { Meta, Story } from '@storybook/react'
+import React from 'react'
+import UnitInput, { UnitInputProps } from '..'
+
+export default {
+  component: UnitInput,
+  parameters: {
+    docs: {
+      description: {
+        component: `UnitInput is a component made out of 2 components: a **TextBox** and a **RichSelect**.`,
+      },
+    },
+  },
+  title: 'Components/Data Entry/UnitInput',
+} as Meta
+
+const Template: Story<UnitInputProps> = args => (
+  <UnitInput name="test" {...args} />
+)
+
+export const Default = Template.bind({})
+Default.parameters = {
+  docs: {
+    storyDescription: `UnitInput provide data as follows:
+
+\`\`\`js
+{
+    unit: [string],
+    value: [string]
+}
+\`\`\``,
+  },
+}
```

Comment:
Can't you do it this way instead:

```suggestion
Default.parameters = {
  docs: {
    storyDescription: "UnitInput provide data as follows:

```js
{
    unit: [string],
    value: [string]
}
```",
  },
}
```

?

Reply:
`"` doesn't work with multiline/markdown